src/components/TeacherComponents/LessonList/LessonList.tsx

```diff
@@ -78,8 +84,33 @@ const LessonList: FC<LessonListProps> = (props) => {
         </>
       ) : null}
       {lessonCount > LESSONS_PER_PAGE ? (
-        <Box $width="100%" $mt={[0, "auto"]} $pb={[30, 44]} $pt={[46, 36]}>
-          <Pagination pageName={unitTitle} {...paginationProps} />
+        <Box
+          $width="100%"
+          $mt={[0, "auto"]}
+          $pb={[30, 44]}
+          $pt={[46, 36]}
+          data-testid="pagination-box"
+        >
+          <OakPagination
+            {...paginationProps}
+            initialPage={currentPage}
```

Comment:
is this the same thing with 2 different names? if so can we rename in one place so it's consistent?

Reply:
Removed `initialPage` prop so component only takes currentPage, no internal regulation of state ✅